UDV-Core/src/Modules/ConsultDoc/DocumentController.js

```diff
@@ -129,9 +131,33 @@ export function DocumentController(view, controls, options = {},config)
         if (this.docIndex > 0 || this.setOfDocuments.length == 0)
         {
             this.docIndex--;
-            var currentDoc = this.getCurrentDoc();
-            return this.getCurrentDoc();
         }
+        return this.getCurrentDoc();
+
+    }
+
+    /**
+     * Reset browser at the begining of documents list
+     */
+    //=============================================================================
+    this.reset = function reset(){
+      this.docIndex = 0;
+      this.documentBrowser.docIndex = 1;
+      this.currentDoc = this.setOfDocuments[0];
+      this.documentBrowser.updateBrowser();
+    }
+
+    //show or hide delete/update button
```

Comment:
Please mention in your comment that these buttons are useful for the contribute extension and that in the future they should be managed by ContributeController

Reply:
ok